source/docs/advanced/features/external-logins.md

```diff
@@ -0,0 +1,314 @@
+---
+id: external-logins
+title: External Logins
+---
+
+<blockquote class="feature--new">
+_Since version 2.13_
+</blockquote>
+
+<blockquote class="feature--new">
+_Beta feature (api may change)_
+</blockquote>
+
+# External Logins
+
+Front Commerce supports external logins. The API for supporting external logins comes in 2 parts:
+* The `LoginProvider`s which implements the login with the external systems (such as Facebook or Google)
+* The `ExternalLoginHandler` which implement the login with your platform (such as Magento1, Magento2, Proximis or BigCommerce).
+
+Out of the box we ship two `LoginProvider`s -the `FacebookProvider` and the `GoogleProvider`- and two `ExternalLoginHandler`s -the `Magento1ExternalLoginHandler` and the `Magento2ExternalLoginHandler`.
```

Comment:
isn't it a bit too detailed for something that may/will change ?

Reply:
i think what would change will be more on the implementaion lavel the concepts of LoginProvider and LoginHandler will remain